spec/factories/orders.rb

```diff
@@ -6,7 +6,7 @@
     user factory: :regular_user
     address factory: :address
     status "Pending"
-
+    shop nil
```

Comment:
@andela-anwocha is there any reason why you are setting the shop field to nil here?